assets/training/general/environments/acpt-pytorch-2.0-cuda11.7/context/Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN /var/requirements/install_system_requirements.sh && \
     rm -f /etc/nginx/sites-enabled/default
 ENV SVDIR=/var/runit
 ENV WORKER_TIMEOUT=400
+ENV DS_ACCELERATOR=cuda
```

Comment:
Can you confirm if this is no longer the issue? https://github.com/microsoft/DeepSpeed/issues/3680

Reply:
Sorry for the late reply. Confirmed not to be causing an issue. Also, reading through the thread, it sounds like the print statement would've been present regardless of whether or not we set the env variable since `ds_accelerator` will be auto-detected. We explicitly set `cuda` as an env variable because it would've been auto-detected as `cpu` otherwise

Reply:
@savitamittal1 Hi, any updates?